auth/src/test/java/com/firebase/ui/auth/test_helpers/ActivityHelperShadow.java

```diff
@@ -45,4 +51,9 @@ public FirebaseAuth getFirebaseAuth() {
     public CredentialsApi getCredentialsApi() {
         return credentialsApi;
     }
+
+    @Implementation
```

Comment:
Ah I think I see why you implemented this method, is it just so you can shadow it easily?